ext/RMagick/rmimage.c

```diff
@@ -9639,7 +9642,6 @@ Image_opaque_channel(int argc, VALUE *argv, VALUE self)
 
     okay = OpaquePaintImageChannel(new_image, channels, &target_pp, &fill_pp, invert);
 
-    // Restore saved fuzz value
```

Comment:
please split PR to remove unnecessary comment.

Reply:
Sure will do.